{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "452bf0d0_bd900945",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 2
      },
      "lineNbr": 0,
      "author": {
        "id": 1329751
      },
      "writtenOn": "2021-12-16T16:34:14Z",
      "side": 1,
      "message": "PTAL.. I\u0027m not sure what extra value this provides, but we could consider running it. It wouldn\u0027t catch the issue I was running into with the retrace because that would only happen after Reset.",
      "revId": "b81206b2bd6a9beacdf3e7f94a741a022bee5fec",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    }
  ]
}